src/main/java/frc/robot/maps/FrancoisMap.java

```diff
@@ -41,10 +41,14 @@ public DifferentialDriveMap getDriveMap() {
 
             @Override
             public SendableSpeedController getRight() {
+                rightLeader.setInverted(true);
```

Comment:
Might be good to comment here that we do this instead of relying upon the differential drive to do this.

Reply:
Fixed

Reply:
Comment doesn't quite explain it

Reply:
Better